app/views/posts/show.html.erb

```diff
@@ -1 +1,13 @@
-<h1>Total Comments</h1>
+<h2>Total Comments</h2>
+<div>
+  <div class="user-card left">
+    <h3><%= @post.title %> by <%= @user.name %></h3>
+    <div class="right">
+        <span class="span-post">Comments: <%= @post.comments.count %></span>
+        <span class="span-post">Likes: <%= @post.likes.count %></span>
+      </div>
+      <br>
+    <p><%= @post.text %></p>
```

Comment:
- [ ] Good job working on views, however, make sure to follow the given layout for the post show. Kindly display all comments for the post show 👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![blog_single_post](https://github.com/mohashyne/Blog_App/assets/57408419/15a3bf27-0dcf-4ffa-8809-91902397ff93)| |![Screenshot 2023-10-05 171019](https://github.com/mohashyne/Blog_App/assets/57408419/3aab8a34-81e9-4883-9e4f-994e78c71734)| |